package.json

```diff
@@ -1,15 +1,27 @@
 {
-  "name": "x-create-shield-study",
+  "name": "create-shield-study",
+  "description": "Create new Shield studies via the CLI",
   "version": "1.0.0",
-  "description": "",
+  "author": "Mozilla",
+  "bin": {
+    "create-shield-study": "create-shield-study"
+  },
+  "bugs": {
+    "url": "https://github.com/mozilla/create-shield-study/issues"
+  },
+  "dependencies": { 
+    "plop": "^2.1.0"},
```

Comment:
This was subtle and took a bit to debug, but for our combined future reference, we need to make the CLI script dependencies non-devDependencies so npx can find them.